### ISSUE TITLE
Getting "unexpected key" github CI error while deploying aws ecs

### ISSUE THREAD

**Issue by allisaurus** [aws-actions/amazon-ecs-deploy-task-definition#164]
Additional fields present in the results of [ecs:DescribeTaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_DescribeTaskDefinition.html) are being validated against, causing failures for users who pipe these results into this action. 

First reported here:
https://github.com/aws-actions/amazon-ecs-render-task-definition/issues/38#issuecomment-767667278

We are currently working on a fix which adds these properties to the list of `IGNORED_TASK_DEFINITION_ATTRIBUTES`

**Comment by allisaurus** [comment]
Fix has been merged, and will be released in the very near future. 

Those effected should be able to point their action workflows at `aws-actions/amazon-ecs-deploy-task-definition@master` to use + test the latest.

**Comment by allisaurus** [comment]
This issue has been addressed in [v1.4.2](https://github.com/aws-actions/amazon-ecs-deploy-task-definition/blob/v1.4.2/CHANGELOG.md)!

Please let us know if you continue to encounter this issue.

**Comment by mlecar** [comment]
Hi @allisaurus , Is there a way to silence these warnings? I am using the file that I registered using ```aws ecs register-task-definition```.
```
Run aws-actions/amazon-ecs-deploy-task-definition@v1.4.2
Warning: Ignoring property 'compatibilities' in the task definition file. This property is returned by the Amazon ECS DescribeTaskDefinition API and may be shown in the ECS console, but it is not a valid field when registering a new task definition. This field can be safely removed from your task definition file.
Warning: Ignoring property 'taskDefinitionArn' in the task definition file. This property is returned by the Amazon ECS DescribeTaskDefinition API and may be shown in the ECS console, but it is not a valid field when registering a new task definition. This field can be safely removed from your task definition file.
Warning: Ignoring property 'requiresAttributes' in the task definition file. This property is returned by the Amazon ECS DescribeTaskDefinition API and may be shown in the ECS console, but it is not a valid field when registering a new task definition. This field can be safely removed from your task definition file.
Warning: Ignoring property 'revision' in the task definition file. This property is returned by the Amazon ECS DescribeTaskDefinition API and may be shown in the ECS console, but it is not a valid field when registering a new task definition. This field can be safely removed from your task definition file.
Warning: Ignoring property 'status' in the task definition file. This property is returned by the Amazon ECS DescribeTaskDefinition API and may be shown in the ECS console, but it is not a valid field when registering a new task definition. This field can be safely removed from your task definition file.
Warning: Ignoring property 'registeredAt' in the task definition file. This property is returned by the Amazon ECS DescribeTaskDefinition API and may be shown in the ECS console, but it is not a valid field when registering a new task definition. This field can be safely removed from your task definition file.
Warning: Ignoring property 'registeredBy' in the task definition file. This property is returned by the Amazon ECS DescribeTaskDefinition API and may be shown in the ECS console, but it is not a valid field when registering a new task definition. This field can be safely removed from your task definition file.
```
